examples/step-15/step-15.cc

```diff
@@ -406,7 +406,7 @@ namespace Step15
     // values. Thus, we have the new values written into a temporary vector,
     // and only afterwards write them into the solution vector object:
     Vector<double> tmp(dof_handler.n_dofs());
-    solution_transfer.interpolate(current_solution, tmp);
```

Comment:
The commentary above this block is no longer accurate and needs to be adjusted.

Reply:
And if we don't need `current_solution` any more, we can probably get rid of the `tmp` vector as well.